javascript-light-bulb/main.js

```diff
@@ -0,0 +1,16 @@
+var $container = document.querySelector('.container');
+var $button = document.querySelector('button');
+var count = 0;
+
+function handleClick(event) {
+  count++;
+  if (count % 2 === 0) {
```

Comment:
A count of the clicks will work here, but you are keeping track of a count and just determining if that count is even or odd, one of two states. A boolean would be a better way to keep track of whether something is one of two different states. You should make a variable to keep track of if the light is currently on, with a boolean value.